style.css

```diff
@@ -0,0 +1,94 @@
+body {
+  margin: 0;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-around;
+}
+
+.toolbar img {
+  margin-top: 25px;
+}
+
```

Comment:
- [ ] The CSS is a bit of the chart you have margin to the right see the screenshot
![nn](https://user-images.githubusercontent.com/61562626/129162075-eaffa608-7d4c-42ed-8854-d02c8c6a4c0e.png)

Reply:
![image](https://user-images.githubusercontent.com/41170585/129169398-507f406d-a7a2-468a-acce-00e69143f25a.png)

Reply:
The issue here is how you are linking the file, I believe it's a capital L that starts the Linkedin , like this `vector/Linkedin_icon.png` not `vector/linkedin_icon.png`